### PR TITLE
Add event entries

### LINKED ISSUE #19: Events: missing entries
https://www.calconnect.org/events/past-events

We are missing entries from the past events page.

### PR DESCRIPTION
**Pull request by MuhammadAhsan252**
close #19

### CODE REVIEW

File: _events/upcoming-virtual-events-planned.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+date: 2024-11-14
+image: /assets/images/CalConnectEventTBD_0.jpg
+---
+
+== Upcoming Virtual Events Planned
+
+image::/assets/images/CalConnectEventTBD_0.jpg[]
+
+CalConnect will host some one or two day virtual events on specific topics. Any suggestions on appropriate topics are welcome. We have currently in mind:
+
```
Comment (ronaldtse) [inline _events/upcoming-virtual-events-planned.adoc:14]: Missing lots of content here:

https://www.calconnect.org/events/upcoming-virtual-events-planned
Screenshot 2024-12-10 at 4 36 59 PM

**Review by ronaldtse — APPROVED** [review]
Lots of information missing on the event pages! You can just implement the tabs as sections on the same page. But all the information needs to be here.

**Review by ronaldtse — CHANGES_REQUESTED** [review]
Need to fill in all the event pages with details.

**Comment by ronaldtse** [comment]
@MuhammadAhsan252 this PR is very very confusing. It seems that you used some AI processing to summarize the content?? We do not want any post-processing. We only want the original data.

**Comment by MuhammadAhsan252** [comment]
@ronaldtse In the latest commits, I only used Coradoc to covert HTML to Asciidoc. Afterwards I did some manual edits required for links and content. Other than that there was no post-processing done.

**Pull request merged**